go/internal/services/keys/verifier.go

```diff
@@ -117,12 +125,12 @@ func (k *KeyVerifier) log() {
 		RequestID:   k.session.RequestID(),
 		WorkspaceID: k.Key.WorkspaceID,
 		Time:        time.Now().UnixMilli(),
-		Region:      "",
 		Outcome:     string(k.Status),
 		KeySpaceID:  k.Key.KeyAuthID,
 		KeyID:       k.Key.ID,
 		IdentityID:  k.Key.IdentityID.String,
 		Tags:        k.tags,
+		Region:      os.Getenv("UNKEY_REGION"),
```

Comment:
It's a small thing, but can we pass this through from the cli args?

1. this might lead to inconsistent logs when the flag is provided via cli, rather than env
2. this might break easily if we rename the env variable, cause it's now used in two places